uwblueprint/project-read-frontend
# [TS migration] Move registration table to Typescript 📦

### Notion ticket link



[TS migration](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=e47e78d6b54e4120aa13f06bb61640c5)



### Implementation description

- migrated `FamilyTable` to TSX, did some refactoring along the way (no logic changes)
- created enums for default field names
- defined types for fields, families, and students



### Steps to test

1. `yarn`
2. `yarn start`
3. verify that the app compiles, and the registration table still appears



### What should reviewers focus on?

- any TS best practices?

### Checklist
